package.json

```diff
@@ -37,6 +37,7 @@
   },
   "devDependencies": {
     "@sentry-internal/eslint-config-sdk": "^6.2.1",
+    "@types/glob": "^7.2.0",
```

Comment:
Is this a `devDependencies`? because we are using it directly in the code.

Reply:
Yes, it is a devDep, it's only typing for the Glob dep. And typings of our deps are used only for development. The actual glob lib is regular dep and install when wizard is used.